From 38301420e0233826de3ab0809c5081dd03ab4101 Mon Sep 17 00:00:00 2001
From: Danct12 <danct12@disroot.org>
Date: Sun, 23 Jun 2024 06:40:42 +0000
Subject: [PATCH 5/7] arm64: dts: rockchip: pinetab2: Add ethernet0 alias to
 BES2600

This allows U-Boot to set local-mac-address property on the device,
allowing static mac address to happen.
---
 arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi b/arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi
index b6f9edc094d5..20995465d42c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi
@@ -13,6 +13,7 @@ / {
 	chassis-type = "tablet";
 
 	aliases {
+		ethernet0 = &bes2600;
 		mmc0 = &sdhci;
 		mmc1 = &sdmmc0;
 	};
-- 
2.45.2

